bundle/edu.gemini.ags/src/test/scala/edu/gemini/ags/gems/GemsResultsAnalyzerSpec.scala

```diff
@@ -71,7 +71,7 @@ class GemsResultsAnalyzerSpec extends MascotProgress with SpecificationLike with
       }
 
       LOGGER.info("gems results: size = " + gemsGuideStars.size)
-      gemsGuideStars should have size 252
+      gemsGuideStars should have size 228
```

Comment:
is this value to make the test pass or was it derived from somewhere?

Reply:
I checked this with my debug filtering as per the terminal screenshot example above to see how many asterisms were being filtered out.